rflx/generator/generator.py

```diff
@@ -47,52 +47,52 @@ def create_internal_functions(
                         "Set_Field_Value",
                         [
                             InOutParameter(["Ctx"], "Context"),
-                            Parameter(["Value"], "Field_Dependent_Value"),
-                            OutParameter(["First", "Last"], self.types.bit_index),
+                            Parameter(["Val"], "Field_Dependent_Value"),
+                            OutParameter(["Fst", "Lst"], self.types.bit_index),
                         ],
                     ),
                     [
                         ObjectDeclaration(
-                            ["F"],
+                            ["First"],
                             self.types.bit_index,
-                            Call("Field_First", [Name("Ctx"), Selected("Value", "Fld")]),
+                            Call("Field_First", [Name("Ctx"), Selected("Val", "Fld")]),
                             True,
                         ),
                         ObjectDeclaration(
-                            ["L"],
+                            ["Last"],
                             self.types.bit_index,
-                            Call("Field_Last", [Name("Ctx"), Selected("Value", "Fld")]),
+                            Call("Field_Last", [Name("Ctx"), Selected("Val", "Fld")]),
```

Comment:
Wouldn't the use of `First`/`Last` as constant names cause another potential for naming conflicts? Maybe we should use something less common here?